src/views/bls12381.rs

```diff
@@ -491,19 +497,25 @@ impl<'a> ConvView for View<'a> {
                 key_share
                     .encode(&mut buf)
                     .map_err(|e| ConversionsError::Ssh(e.into()))?;
+                    .map_err(|e| ConversionsError::SshEncoding(e))?;
+                    .map_err(ConversionsError::SshEncoding)?;
```

Comment:
same.

Reply:
Fixed as of https://github.com/cryptidtech/multikey/pull/2/commits/f479bd484b9c377a0a44684ec0b4160d04db1b71